src/status_im2/contexts/quo_preview/selectors/disclaimer.cljs

```diff
@@ -1,27 +1,49 @@
 (ns status-im2.contexts.quo-preview.selectors.disclaimer
   (:require [quo2.components.buttons.button :as button]
-            [quo2.components.selectors.disclaimer.view :as quo]
+            [quo2.components.selectors.disclaimer.view :as disclaimer]
             [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]
             [react-native.core :as rn]
-            [reagent.core :as reagent]))
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "Checked:"
+    :key   :checked?
+    :type  :boolean}
+   {:label "Blur (only for dark theme):"
+    :key   :blur?
+    :type  :boolean}])
 
 (defn cool-preview
   []
-  (let [checked? (reagent/atom false)]
+  (let [state (reagent/atom {:checked? false
+                             :blur?    true})]
     (fn []
-      [rn/view
-       {:margin-bottom      50
-        :padding-vertical   16
-        :padding-horizontal 20}
-       [rn/view
-        {:padding-vertical 60
-         :align-items      :center}
-        [quo/view
-         {:container-style {:margin-bottom 40}
-          :on-change       #(swap! checked? not)}
-         "I agree with the community rules"]
-        [button/button
-         {:disabled (not @checked?)}
+      [rn/view {:style {:flex 1}}
+       [rn/view {:style {:flex 1}}
+        [preview/customizer state descriptor]]
+       [rn/view {:style {:padding-horizontal 15}}
+        (when (and (:blur? @state) (theme/dark?))
+          [rn/view
+           {:style {:position :absolute
+                    :top      0
+                    :bottom   0
+                    :left     0
+                    :right    0}}
+           [preview/blur-view
+            {:style                 {:flex        1
+                                     :align-items :center}
+             :show-blur-background? true}]])
+        [rn/view
+         {:style {:margin-vertical 50
+                  :width           "100%"}}
+         [disclaimer/view
+          {:blur?     (:blur? @state)
+           :checked?  (:checked? @state)
+           :on-change #(swap! state update :checked? not)}
+          "I agree with the community rules"]]
+        [button/button {:disabled (not (:checked? @state))}
```

Comment:
Instead of adding :disabled as false
We can use `cond-> {}` and only add the key when the checkbox is not checked

Reply:
I really like the use of  `cond->` :smile: 

But this time I think it creates more complex code than requied, I mean, it looks shorter and simpler:

```clojure
[button/button {:disabled (not (:checked? @state))}
  "submit"]
```

than:

```clojure
[button/button
  (cond-> {}
   (:checked? @state) (assoc :disabled true))
  "submit"]
```

wdyt?

Reply:
I vote keep it simple 👍

Reply:
I think it's better to not have the disabled property at all if it's no true
But I guess it's fine because the button is expected to be disabled and then enabled